MSBuild.sln

```diff
@@ -82,6 +82,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Build.UnitTests.S
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Build.BuildCheck.UnitTests", "src\BuildCheck.UnitTests\Microsoft.Build.BuildCheck.UnitTests.csproj", "{434CC6DB-1E66-4FB1-A66C-D5BBE99F0ED8}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Analyzer.Templates", "template_feed\Microsoft.Analyzer.Templates.csproj", "{A86EE74A-AEF0-42ED-A5A7-7A54BC0773D8}"
```

Comment:
@JanKrivanek , isn't it better to use more generic name for the templates?

Reply:
I have the opposite opinion, I think this name is too generic. I'd prefer if we specified that the analyzer template is for BuildCheck, otherwise it might get confused with other analyzers in Microsoft projects.

Reply:
Thank you for the clarification, I was not aware about the level package.

Reply:
@YuliiaKovalova Do you have any ideas how to name it more generic？

Reply:
I'd suggest `Microsoft.Build.Templates` to align with the existing `Microsoft.Build.*` packages.